docs/src/quickstart.md

```diff
@@ -107,66 +166,130 @@ processing" communities.
 Because images are just arrays, some environments (e.g.,
 Juno or IJulia/Jupyter) will display numeric arrays as arrays using a text
 representation but will display 2d arrays that have `Colorant`
-elements as images.  You can "convert" in the following ways:
+elements as images.
+
+## Color conversions are construction/view
+
+Conversions between different `Colorant`s are straightforward:
+
+```@repl pixel
+RGB.(img_gray) # Gray => RGB
+Gray.(img_rgb) # RGB => Gray
+```
+
+!!! note
+    You'll see [broadcasting](https://docs.julialang.org/en/v1/manual/arrays/#Broadcasting-1)
+    semantics used in JuliaImages here and there, check the documentation if
+    you're not familiar with it.
 
-```jldoctest
-using Images
+Sometimes, to work with other packages, you'll need to convert a ``m \times n``
```

Comment:
a ``m \times n`` `RGB` image to an ``m \times n \times 3``  image and vice versa.